brunton_lab_to_nwb/brunton_widgets.py

```diff
@@ -56,32 +56,23 @@ def trials_psth(self, before=0., after=1., figsize=(12, 12)):
         """
         starts = self.events - before
         stops = self.events + after
-        # Construct time vector
-        # tt_start = self.spatial_series.starting_time
-        # num_pts = np.shape(self.spatial_series.data[:])[0]
-        # tt_stop = num_pts / self.spatial_series.rate
-        # tt = np.linspace(tt_start, tt_stop, int((num_pts)))
-        # # Compute speed
-        # speed = compute_speed(self.spatial_series.data[:], tt)
-        # speed = np.reshape(speed, (-1, 1))
-        # # Append to position data
-        # pos_Wspeed = np.append(self.spatial_series.data[:], speed, axis=1)
-        # del self.spatial_series.data
-        # self.spatial_series.data = pos_Wspeed
 
         trials = align_by_times(self.spatial_series, starts, stops)
+        zero_ind = before * self.spatial_series.rate
```

Comment:
this will not work if `timestamps` are used. Look into https://github.com/NeurodataWithoutBorders/nwb-jupyter-widgets/blob/daefda2a695f76a7a787d78de003ddc6a2a53482/nwbwidgets/utils/timeseries.py#L7 and `np.searchsorted`